pyirf/tests/test_integration.py

```diff
@@ -0,0 +1,50 @@
+import glob
+import subprocess
+import logging
+import os
+from astropy.io import fits
+from ogadf_schema.irfs import AEFF_2D, EDISP_2D, PSF_TABLE, BKG_2D, RAD_MAX
+
+
+def test_integration_test(caplog):
+
+    caplog.set_level(logging.INFO)
+
+    ROOT_DIR = os.path.dirname(os.path.abspath("setup.py"))
+
+    for example_script in glob.glob(f"{ROOT_DIR}/examples/calculate_*.py"):
```

Comment:
I don't like having this globbing here. We should have a single test function for each integration tests.

This makes it also clearer which function was executed.